commit f190ff558f814ed65575ca82e3a1fe98dfc410df (refs/changes/87/387/1)
Author: David Pacheco <dap@joyent.com>
Date:   2016-08-30T13:27:50-07:00 (3 years, 1 month ago)
    
    MANTA-2974 manta-factoryreset crashes when no "manta" application is present

diff --git a/cmd/manta-factoryreset.js b/cmd/manta-factoryreset.js
index b9aa480..1ef99e2 100755
--- a/cmd/manta-factoryreset.js
+++ b/cmd/manta-factoryreset.js
@@ -192,6 +192,10 @@ async.waterfall([
 		var log = self.log;
 		log.info('checking stage');
 
+		if (!self.application) {
+			return (cb(null));
+		}
+
 		var app = self.application;
 
 		if (app.metadata.SIZE === 'production') {
